action/protocol/staking/contractstaking/indexer.go

```diff
@@ -82,3 +105,110 @@ func (s *Indexer) BucketTypes() ([]*BucketType, error) {
 	}
 	return bts, nil
 }
+
+// PutBlock puts a block into indexer
+func (s *Indexer) PutBlock(ctx context.Context, blk *block.Block) error {
```

Comment:
make sure the indexer-related files are located in `/blockindex`

Reply:
moved